lms/static/js/fixtures/edxnotes/toggle_notes.html

```diff
@@ -0,0 +1,6 @@
+<div class="action-inline edx-notes-visibility">
+    <a href="#" class="action-toggle-notes is-disabled is-active" role="button" aria-pressed="">
+        <i class="checkbox-icon icon-check"></i>
```

Comment:
Unfortunately you have to update all your icons to use the new font-awesome scheme (fa-check). If you rebase you'll find that all these icons are broken, as the new version of font-awesome was merged last week.